go/client/cmd_dump_keyfamily.go

```diff
@@ -23,12 +23,27 @@ func indentSpace(level int) string {
 
 type CmdDumpKeyfamily struct {
 	libkb.Contextified
+	user string
+}
+
+func (v *CmdDumpKeyfamily) ParseArgv(ctx *cli.Context) error {
+	nargs := len(ctx.Args())
+	if nargs > 1 {
+		return fmt.Errorf("dump-keyfamily only takes on argument, the user to lookup")
+	}
+	if nargs == 1 {
+		v.user = ctx.Args()[0]
+	}
+	return nil
 }
 
 func NewCmdDumpKeyfamily(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
 	return cli.Command{
-		Name:  "dump-keyfamily",
-		Flags: []cli.Flag{},
+		Name:         "dump-keyfamily",
+		ArgumentHelp: "[username]",
+		Usage:        "Print out a user's current key family",
+		Description:  "Print out a user's current key family. Don't specify a username to dump out your own keys.",
```

Comment:
In my head "Don't specify a username..." sounds like it's a warning :) Maybe "with no argument, prints your own key family."